tests/test_smoketest.py

```diff
@@ -52,25 +52,28 @@ def smoketest(self, starting_replica, checkout_bucket, test_bucket):
             'query': {'match': {'files.cell_suspension_json.biomaterial_core.biomaterial_id': biomaterial_id}}
         }
         tombstone_query = {"query": {"bool": {"must": [{"term": {"admin_deleted": "true"}}]}}}
-        queries = [bundle_query, tombstone_query]
+        tombstone_query_jmes = "event_type=='CREATE'"
+        queries = [(bundle_query, "elasticsearch"), (tombstone_query, "elasticsearch"),
+                   (tombstone_query_jmes, "jmespath")]
```

Comment:
Why isn't there a bundle query for JMESPath?

Reply:
ah, it got lost; ill find it.